src/main.ts

```diff
@@ -9,7 +9,7 @@ const PORT = process.env.PORT;
 
 async function bootstrap() {
   const app = await NestFactory.create<NestExpressApplication>(AppModule);
-
+  app.enableCors();
```

Comment:
We discussed how this was needed, because locally requests are made to port 4200, but the backend server is at 3000, so enabling requests from other origins is needed. 👍